swig/python/codegen/__init__.py

```diff
@@ -0,0 +1,43 @@
+# coding: utf-8
+from six.moves import cStringIO
```

Comment:
We should not need to use `six`, a python 2 / 3 compatibiliity library, given we explicitly only support python 3.7+

You should use [`io.StringIO`](https://docs.python.org/3/library/io.html#io.StringIO) instead (based on the `six` docs).

Reply:
Killed all references to six